### DISCUSSION TITLE
Does assimp support the Standard Surface in Maya?

### DISCUSSION THREAD

**Discussion by KK-5** [assimp/assimp#4650] [Q&A]
I imported the FBX that exported from Maya with assimp 5.2.1 ,then I found that the Standard Surface material could not be resolved，the Diffuse Color and the Base Color is empty,  I was sure the color was set in the FBX. Does assimp support the Standard Surface in Maya? the lambert and phong material is OK.

**Comment by kimkulling** [comment]
I have never tested this. Do you have a spec for this kind of material-definition?

**Comment by KK-5** [comment]
I do not have the spec, but I found the material-definition format like this in FBX
![image](https://user-images.githubusercontent.com/55627654/187012015-e8062abe-27a3-42e2-8925-d6b24a047558.png)
